.gitignore

```diff
@@ -0,0 +1 @@
+text.md
```

Comment:
- [ ] After you install linters on your local machine, a node_modules folder is generated. That file should also be added to the .gitignore file. Kindly add it.

Reply:
hi thanks i have done most of the requirements, but where can i find the node-modules, im really sorry for this question but this is my first time using the linters